firefox-ios/Ecosia/Account/Auth/Auth0SessionTokenRequest.swift

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+/// Struct for encoding the JSON request body
+struct TokenRequest: Codable {
+    let grantType: String
+    let subjectToken: String
+    let subjectTokenType: String
+    let requestedTokenType: String
+    let clientId: String
+}
```

Comment:
Maybe it would make sent to move those inside `Auth0SessionTokenRequest` and directly call `Body`? Would follow the same pattern as `SingularConversionValueRequest` (even if there those were query parameters).

Reply:
I like that 👍 .
I had to make some changes to make it successfully applicable but its nicer and justifies the `data: Data` definition.